lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/region_event_card.dart

```diff
@@ -0,0 +1,119 @@
+            ))
+        .toList();
+
+    final stackLayers = List<Widget>.generate(items.length, (index) {
+      return Padding(padding: EdgeInsets.fromLTRB(index.toDouble() * overlap, 0, 0, 0), child: items[index]);
+    });
+
+    return Stack(children: stackLayers);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (_) => RegionEventCardBloc()..add(OnLoadRegionEventMembers(regionEvent.users)),
+      child: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 8.0),
+        child: InkWell(
+          borderRadius: BorderRadius.circular(16.0),
+          child: Container(
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(16.0),
+              border: Border.all(color: AppColors.grey1),
+            ),
+            child: Padding(
+              padding: const EdgeInsets.all(4.0),
+              child: Column(
+                children: [
+                  ClipRRect(
+                    borderRadius: BorderRadius.circular(10.0),
+                    child: CachedNetworkImage(
+                      imageUrl: regionEvent.eventImage,
+                      fit: BoxFit.cover,
+                      height: MediaQuery.of(context).size.width - 250,
+                      width: MediaQuery.of(context).size.width - 32,
+                      placeholder: (_, __) => const SizedBox.shrink(),
+                      errorWidget: (_, __, ___) => const SizedBox.shrink(),
+                    ),
+                  ),
+                  const SizedBox(height: 12.0),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Text(
+                        regionEvent.eventName,
+                        style: Theme.of(context).textTheme.buttonLowEmphasis,
+                        overflow: TextOverflow.ellipsis,
+                      ),
+                      Text(DateFormat.yMMMMd().format(DateTime.parse(regionEvent.createdTime.toDate().toString()))),
+                    ],
+                  ),
+                  const SizedBox(height: 8.0),
+                  Row(
+                    children: [
+                      BlocBuilder<RegionEventCardBloc, RegionEventCardState>(
+                        builder: (context, state) {
+                          return state.profiles.isNotEmpty
+                              ? _buildMemberImages(state.profiles)
+                              : const SizedBox.shrink();
+                        },
+                      ),
+                      const SizedBox(width: 6.0),
+                      Text(
+                        '${context.loc.regionMainMembersTitle(regionEvent.users.length, regionEvent.readableMembersCount)} ${context.loc.regionEventCardJoinedTitle}',
+                        style: Theme.of(context).textTheme.subtitle3.copyWith(color: AppColors.grey2),
+                      ),
```

Comment:
@gguijarro-c-chwy I really think we need improve our textstyle system.
We don't need fancy names from the desingner when all we need is to name a TextStyle is:

- FontWeight
- FontSize
- Color

Maybe we could add a style like normal or italic but i don't see any of them in the project.
Much easy to understand, access and avoid duplicates.

Reply:
Anyway are we using a specific font in the project? like Open Sans, Roboto, etc??

Reply:
Agree, the problem is, we dont have a designer.

Reply:
We should go over our text styles and recreate them, and anyone that comes to design for this app, must use those styles